core/templates/dev/head/pages/exploration-editor-page/statistics-tab/issues/answer-submit-action.directive.html

```diff
@@ -0,0 +1,10 @@
+<[$ctrl.actionIndex]>. Submitted answer "
+<span ng-if="$ctrl.currentStateName == $ctrl.destStateName">
+  <angular-html-bind html-data="$ctrl.getShortAnswerHtml()"></angular-html-bind>
+  " in card "<[$ctrl.currentStateName]>".
+</span>
+<span ng-if="$ctrl.currentStateName != $ctrl.destStateName">
+  <angular-html-bind ng-if="$ctrl.currentStateName != $ctrl.destStateName" html-data="$ctrl.getShortAnswerHtml()"></angular-html-bind>
```

Comment:
I think you can get rid of the ng-if here, and move the angular-html-bind out of the ng-ifs into the shared part of the code. Am I missing something?

Reply:
Oh right. Done.